lib/oauthutil/oauthutil.go

```diff
@@ -24,6 +25,11 @@ import (
 	"golang.org/x/oauth2"
 )
 
+var (
+	TemplatePath string
```

Comment:
You should add a comment to this exported variable, which starts with the word "TemplatePath". The linter will complain, when I enable the workflow to run..